src/lib/utils.js

```diff
@@ -50,10 +50,9 @@ export function debounce(fn, delay) {
 	};
 };
 
-const itemHtml = document.createElement('div');
-itemHtml.className = 'sv-item-content';
-
 export function highlightSearch(item, isSelected, $inputValue, formatter) {
+  const itemHtml = document.createElement('div');
+  itemHtml.className = 'sv-item-content';

```

Comment:
The `itemHtml` needs to be defined only once, no matter how many Svelecte components are being used. Can you implement it this way?

```suggestion
if (!itemHtml) {
  itemHtml = document.createElement('div');
  itemHtml.className = 'sv-item-content';
}

```

Reply:
The issue is that you can't reference `document.createElement` at all, because it's being generated server-side. Why does `itemHtml` need to be defined only once? As far as I can tell it's only referenced in highlightSearch. Is it a performance issue? I can probably find a way to call document.createElement fewer times if so.

Reply:
Yes, I did it this way because of _possible_ performance issues with long list, but basically if that should be a concern, `virtualList` should be enabled. (this was implemented before virtual list support has been added)

Reply:
OK, well in that case, the latest commit should help. It's about the best I could do with the limitations of SSR. The idea is that in most cases (hopefully all but I'm not sure), this condition will apply in SSR:

```
  if ($inputValue == '' || item.isSelected) return itemHtmlText;
```

The code that follows is not SSR-safe, but hopefully it will never get there.

Reply:
Yes, it shouldn't because it's only about text highlighting while typing. Thanks for your effort!